ui/eureka/eureka.css

```diff
@@ -293,6 +304,10 @@
   right: 0;
 }
 
+.aspect-ratio-southhorn #label-container {
+  top: 70%;
```

Comment:
I had some issues with this causing the box to clip off the bottom of the overlay unless I sized the overlay above about 550x550px, which is a pretty big size to have to scale up to if the user is running at 1080p. This is probably also an issue with the other zones, but I never really used this overlay for them so I'm not sure, and I didn't get a chance to play around with this before I had to stop OC for raid tonight. Does `bottom: 0px;` not work for this?